src/util/autoclose.ts

```diff
@@ -10,26 +10,12 @@ const isContainedIn = (element: HTMLElement, array?: HTMLElement[]) =>
 const matchesSelectorIfAny = (element: HTMLElement, selector?: string) =>
     !selector || closest(element, selector) != null;
 
-// we have to add a more significant delay to avoid re-opening when handling (click) on a toggling element
-// TODO: use proper Angular platform detection when NgbAutoClose becomes a service and we can inject PLATFORM_ID
-const isMobile = (() => {
```

Comment:
Hmm, are you aware that autoClose is used not only by the tooltip, but also datepicker, popover, dropdown and typeahead?

Removing this will break all others.

Also changes to this code should be tested on Desktop, Android, iOS and iPadOS

Reply:
Currently I don't have access to these browsers. I will have to find a solution on that and update this PR in the future.

Reply:
@maxokorokov Seems that browserstack.com is free for open source projects. `ng-bootstrap` has some e2e tests. Can't we setup `protractor.conf.js` to run against selenium grid provided by `browserstack` ?

Reply:
> Removing this will break all others.
Seems this already is broken on IPad  IOS  > 13, I tested some of the IPads though available on BrowserStack.
You can't close datepickers there.